src/components/BookContainer.js

```diff
@@ -0,0 +1,21 @@
+import { Route, Routes } from 'react-router-dom';
+import React from 'react';
+import Navbar from './Navbar';
+import './Styles/BookContainer.css';
+import Catagory from './Pages/Categories';
+import Book from './Books/Book';
+
+const BookContainer = () => (
+  <div className="container">
+    <Navbar />
+    <Routes>
+      <Route path="/" element={<Book />} />
+      ;
+      <Route path="/category" element={<Catagory />} />
+      ;
+    </Routes>
+
+  </div>
```

Comment:
- [ ] the following codes are supposed to be placed in the `index.js` file so that your codes become structured in a way that **they are scalable**. Normally routes are supposed to be in the index file 😃 👍